testutil/docker.go

```diff
@@ -19,7 +19,10 @@ func IsDockerCase(testName string) bool {
 // SkipDockerIfUnsupported skips the test if the platform is not Linux or the
 // SKIP_DOCKER_TESTS environment variable is explicitly set.
 func SkipDockerIfUnsupported(t *testing.T) {
-	skip, _ := strconv.ParseBool(os.Getenv("SKIP_DOCKER_TESTS"))
+	skip, err := strconv.ParseBool(os.Getenv("SKIP_DOCKER_TESTS"))
```

Comment:
Should we add a `require.NoError(err)` here?

Reply:
This is an optional environment variable for which the default is "false" (i.e. run the docker tests), so we don't   check the error when it's not set.